src/server.h

```diff
@@ -2153,6 +2161,7 @@ int handleClientsWithPendingWritesUsingThreads(void);
 int handleClientsWithPendingReadsUsingThreads(void);
 int stopThreadedIOIfNeeded(void);
 int clientHasPendingReplies(client *c);
+int localConnection(client *c);
```

Comment:
minor preference for:
```suggestion
int isLocalConnection(client *c);
```
or
```suggestion
int isLocalClient(client *c);
```